docs/user-guide.md

```diff
@@ -37,13 +37,18 @@ This guide provides detailed instructions on how to import and configure your Li
 
     - Liberty Maven Plugin -> 3.7.1
 
-    - Liberty Gradle Plugin -> 3.5.1
```

Comment:
Noting that this change to recommended minimum is to support using Gradle 8.x, which did not work with LGP 3.5.x